### PR TITLE
Fixing linter issues for SorrTask693_improve_hpandas_compare_dfs

### PR DESCRIPTION
**Pull request by tkpratardan**
@gpsaggese @sonaalKant @samarth9008 
This is a stacked PR to merge the changes occurring by running linter on the files in #717 and simplify the workflow for the main PR. Please review.

FYR:
The workflow for https://github.com/sorrentum/sorrentum/issues/682 is organized as follows:

1. Main PR: https://github.com/sorrentum/sorrentum/pull/717 (changes to the main function)
2. Stacked PR1: https://github.com/sorrentum/sorrentum/pull/733 (Scaffolding tests for hpandas.compare_dfs())
3. Stacked PR2: https://github.com/sorrentum/sorrentum/pull/726 (fixing linter changes in the files being affected, viz., helpers/hpandas.py and helpers/test/test_hpandas.py )

### CODE REVIEW

File: helpers/test/test_hpandas.py [hunk 1]
```diff
@@ -3654,28 +3657,25 @@ def test5(self) -> None:
...
         """
-        Helper function to get test multi-index dataframe.
-        Example of dataframe returned when `index_is_datetime = True`.
-                                            column1     column2
-        index   timestamp
-        index1  2022-01-01 21:00:00+00:00   -0.122140   -1.949431
-                2022-01-01 21:10:00+00:00   1.303778    -0.288235
-        index2  2022-01-01 21:00:00+00:00   1.237079    1.168012
-                2022-01-01 21:10:00+00:00   1.333692    1.708455
+        Helper function to get test multi-index dataframe. Example of dataframe
```
Comment (sonaalKant) [inline helpers/test/test_hpandas.py:3667]: Can you add ``` around the code block linter has messed it up.

File: helpers/test/test_hpandas.py [hunk 2]
```diff
@@ -3654,28 +3657,25 @@ def test5(self) -> None:
...
+        returned when `index_is_datetime = True`. column1     column2 index
+        timestamp index1  2022-01-01 21:00:00+00:00   -0.122140   -1.949431
+        2022-01-01 21:10:00+00:00   1.303778    -0.288235 index2  2022-01-01
+        21:00:00+00:00   1.237079    1.168012 2022-01-01 21:10:00+00:00
+        1.333692    1.708455.
 
         Example of dataframe returned when `index_is_datetime = False`.
-                            column1     column2
```
Comment (sonaalKant) [inline helpers/test/test_hpandas.py:3673]: Same here.

**Review by sonaalKant — CHANGES_REQUESTED** [review]
LGTM. Some nits .

**Review by gpsaggese — COMMENTED** [review]
After changes suggested by @sonaalKant ok to merge.

BTW @tkpratardan always make sure to review your own PR to spot small problems (even if it's just a linter run).

**Review by samarth9008 — CHANGES_REQUESTED** [review]
TODO(Samarth): Don't merge before creating a PR on cmamp and testing it over there also

**Comment by tkpratardan** [comment]
> After changes suggested by @sonaalKant ok to merge.
> 
> BTW @tkpratardan always make sure to review your own PR to spot small problems (even if it's just a linter run).

Sure @gpsaggese, and @sonaalKant. I am sorry for missing it out. Will make sure to be vigilant.

File: helpers/test/test_hpandas.py (hunk 1 above)
Comment (tkpratardan) [reply helpers/test/test_hpandas.py:3667]: @sonaalKant 
I am not sure  I understand how to implement this. Can you please elaborate? If I understand it right, does it fence the docstrings from being modified by linting operation?

Comment (sonaalKant) [reply helpers/test/test_hpandas.py:3667]: Yes, exactly.

**Review by sonaalKant — APPROVED** [review]

**Review by gpsaggese — COMMENTED** [review]
Good work

**Pull request merged**